ui/js/app.js

```diff
@@ -1223,7 +1218,7 @@ function setExchangeRate(_rates) {
 
   if (rates.cashOut) {
     var cashOut = new BigNumber(rates.cashOut);
-    var cashOutCryptoToFiat = cashOut && formatCrypto(cashOut.round(3).toNumber());
+    var cashOutCryptoToFiat = cashOut && formatCrypto(cashOut.round(3).toNumber(), 2);
```

Comment:
See on line 1216 how we do it for the cash-in, we should the same here. I.E. replace formatCrypto with formatFiat and add the second parameter "2".

Reply:
Changing only this results still results in 3 decimal points, with a trailing zero, due to the way the formatNumber, which is called by formatCrypto, is written